.CI/create_feedstocks.py

```diff
@@ -138,7 +138,7 @@ def write_token(name, token):
     if removed_recipes:
         subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
         msg = ('Removed recipe{s} ({}) after converting into feedstock{s}.'
```

Comment:
I guess it reads better in the next line. I just need to add a space.  Not sure if `feedstocks.[skip appveyor]` will be understood.